Fritz.StreamTools.sln

```diff
@@ -1,46 +1,42 @@
-﻿
```

Comment:
Really don't know why this file changed. Maybe something to do with VS preview updates